nltk/test/sentiment.doctest

```diff
@@ -168,7 +168,7 @@ Vader
     Today SUX!
     compound: -0.5461, neg: 0.779, neu: 0.221, pos: 0.0,
     Today kinda sux! But I'll get by, lol
-    compound: 0.2228, neg: 0.195, neu: 0.531, pos: 0.274,
+    compound: 0.5249, neg: 0.138, neu: 0.517, pos: 0.344,
```

Comment:
looks like another bug -- only `but` or `BUT` was caught before. it makes sense that fixing this bug would make the sentiment more positive